examples/README.md

```diff
@@ -1,14 +1,39 @@
 # Examples
 
-This directory contains examples of how to author durable orchestrations using the Durable Task Python SDK.
+This directory contains examples of how to author durable orchestrations using the Durable Task Python SDK. There are two backends that are compatible with the Durable Task Python SDK: The Dapr sidecar, and the Durable Task Scheduler (DTS)
 
-## Prerequisites
+## Prerequisites for using Dapr
 
 All the examples assume that you have a Durable Task-compatible sidecar running locally. There are two options for this:
 
 1. Install the latest version of the [Dapr CLI](https://docs.dapr.io/getting-started/install-dapr-cli/), which contains and exposes an embedded version of the Durable Task engine. The setup process (which requires Docker) will configure the workflow engine to store state in a local Redis container.
 
-1. Clone and run the [Durable Task Sidecar](https://github.com/microsoft/durabletask-go) project locally (requires Go 1.18 or higher). Orchestration state will be stored in a local sqlite database.
+2. Clone and run the [Durable Task Sidecar](https://github.com/microsoft/durabletask-go) project locally (requires Go 1.18 or higher). Orchestration state will be stored in a local sqlite database.
+
+
+## Prerequisites for using DTS
```

Comment:
These instructions apply only to one of the several examples. I suggest we put DTS-specific instructions and DTS-specific samples in a sub-directory under `examples`. For example, `/examples/dts/activity_sequence_dts.py`, etc.